src/styles/components/_Footer.scss

```diff
@@ -38,8 +38,16 @@
     }
   }
 
-  &::after {
-    content: '·';
+  &:not(:last-child)::after {
+    --marker-size: 2px;
+    background-color: currentColor;
+    border-radius: 50%;
+
+    content: '';
+    display: inline-block;
+    height: var(--marker-size);
+    vertical-align: middle;
+    width: var(--marker-size);
```

Comment:
Do we know what these CSS variables turn into when compiled? For consistency, we may want to use Sass variables. I think this would be the first time we'd be introducing CSS variables, and we wouldn't want to break compatibility for anything with a bugfix.

Reply:
I think the only big difference between the two would be CSS props are modifiable by JS and Sass vars aren't, but I've gone ahead and refactored to use Sass since that's what the project is using.